snmp/datadog_checks/snmp/data/profiles/_base_cisco_voice.yaml

```diff
@@ -45,6 +45,7 @@ metrics:
         OID: 1.3.6.1.4.1.9.9.590.1.6.1.1.2
 
   - MIB: CISCO-CVP-MIB
+    forced_type: gauge
     symbol:
       name: ccvpLicAggMaxPortsInUse
```

Comment:
Did we check if any customers are monitoring these metrics? Would forcing the type change cause any issues to monitors or dashboards?

Reply:
I think we assume they are recent enough that it makes more sense like this.